packages/special-pages/package.json

```diff
@@ -23,7 +23,7 @@
     "@playwright/test": "^1.36.2",
     "http-server": "^14.1.1",
     "@duckduckgo/messaging": "*",
-    "esbuild": "^0.18.19",
```

Comment:
Can this just inherit instead?

Reply:
yeah I think we can move to top-level and then workspaces will make it work for special pages too :)